tests/scripts/ncp_mode

```diff
@@ -213,8 +215,35 @@ otbr_exec_expect_script()
     done
 }
 
+parse_args()
+{
+    BUILD_OT_SIM=1
+    RUN_ALL_TESTS=1
+
+    while [ $# -gt 0 ]; do
+        case $1 in
```

Comment:
```suggestion
        shift
        case $1 in
```

Reply:
Added the `shift` after `case ... esac`.